src/VirtoCommerce.Platform.Core/Security/UserOptionsExtended.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace VirtoCommerce.Platform.Core.Security
+{
+    public class UserOptionsExtended
+    {
+        /// <summary>
+        /// Max. valid time for a password. Property that has the null as default value. Null value means what the password never expires.
+        /// </summary>
+        public TimeSpan MaxPasswordAge { get; set; }
```

Comment:
Please make TimeSpan nullable if you decided to ignore my recommendation to name this setting as MaxPasswordAgeInDays